site/docs/concepts/web-app.md

```diff
@@ -136,15 +126,21 @@ You can proceed to the materialization, or opt to exit to a different page of th
 ## Collections page
 
 The **Collections** page shows a read-only table of [collections](./collections.md) to which you have access.
-The table has many of the same features as the **Captures** table, with several important distinctions
+The table has nearly all of the same features as the **Captures** table, with several important distinctions
 that are called out in the image below.
 
 You can use the table to view each collection's specification and see a sample of its data.
 This can help you verify that collection data was captured as expected and that you'll be able to materialize it how you want, and troubleshoot it necessary.
```

Comment:
Well, this is a historical typo that is MY fault, but... should say "troubleshoot IF necessary"